modules/microshift-kubeconfig-overview.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift_configuring/microshift-kubeconfig.adoc
+
+:_content-type: CONCEPT
+[id="kubeconfig-files_{context}"]
+= Kubeconfig files
+
+The two categories of `kubeconfig` files used in {product-title} are local access and remote access. Every time {product-title} starts, a set of `kubeconfig` files for local and remote access to the API server are generated. These files are generated in the `/var/lib/microshift/resources/kubeadmin/` directory using preexisting configuration information.
+
+Each access type requires a different authentication certificate signed by different Certificate Authorities (CAs). The generation of multiple `kubeconfig` files accommodates this need.
+
+You can use the appropriate `kubeconfig` file for the access type needed in each case to provide authentication details. The contents of {product-title} `kubeconfig` files are determined by either default built-in values or a `config.yaml` file.
+
+[NOTE]
+====
+A `kubeconfig` file must exist for the cluster to be accessible. The values are applied from built-in default values or a 'config.yaml', if one was created.
+====
+
+.Example contents of the kubeconfig files
+[source,bash]
```

Comment:
IIRC, we don't use `[source,bash]` for some reason. This should probably be `[source,terminal]` or `[source,text]` ([relevant section of the contrib guide](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output))